source/crud/tests/unified/client-bulkWrite-replaceOne-sort.yml

```diff
@@ -0,0 +1,76 @@
+description: client bulkWrite updateOne-sort
+
+schemaVersion: "1.4"
+
+runOnRequirements:
+  - minServerVersion: "8.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name crud-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+
+_yamlAnchors:
+  namespace: &namespace "crud-tests.coll0"
```

Comment:
Thanks for using the convention from [Top-level Fields](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.md#top-level-fields) here.